src/main/java/org/terracotta/build/plugins/OsgiPackagePlugin.java

```diff
@@ -0,0 +1,32 @@
+package org.terracotta.build.plugins;
+
+import aQute.bnd.osgi.Constants;
+import aQute.bnd.version.MavenVersion;
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar;
+import org.gradle.api.Action;
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+import org.terracotta.build.plugins.packaging.OsgiManifestJarExtension;
+import org.terracotta.build.plugins.packaging.PackageInternal;
+import org.terracotta.build.plugins.packaging.PackagingExtension;
+import org.terracotta.build.plugins.packaging.PackagingExtensionInternal;
+
+public class OsgiPackagePlugin implements Plugin<Project> {
```

Comment:
Separate the Package and OsgiPackage plugin code so that it's easier to produce non-OSGi packages.